Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -485,7 +507,7 @@ public List<Order> GetOpenOrders(Func<Order, bool> filter = null)
         /// <summary>
         /// Primary thread entry point to launch the transaction thread.
         /// </summary>
-        public void Run()
+        protected void Run()
```

Comment:
This comment is for line 514

VERY COOL! When reading the top I was wondering how it was going to work for the non-threaded version, i.e, how does it break the loop, but duh! it breaks the loop when it's done processing. Gorgeous 👍